manifests/init.pp

```diff
@@ -174,8 +174,6 @@
   Stdlib::Absolutepath $pulpcore_postgresql_ssl_cert = '/etc/pki/katello/certs/pulpcore-database.crt',
   Stdlib::Absolutepath $pulpcore_postgresql_ssl_key = '/etc/pki/katello/private/pulpcore-database.key',
   Stdlib::Absolutepath $pulpcore_postgresql_ssl_root_ca = '/etc/pki/tls/certs/ca-bundle.crt',
-  Array[Stdlib::Absolutepath] $pulpcore_allowed_import_path = $foreman_proxy_content::params::pulpcore_allowed_import_path,
-  Array[Stdlib::Absolutepath] $pulpcore_allowed_export_path = $foreman_proxy_content::params::pulpcore_allowed_export_path,
```

Comment:
Removed these lines because it wouldn't allow me to reassign these variables later.  Also it wouldn't allow me to _not_ assign them here and then assign them later.  But now, Github checks are failing with
```
manifests/init.pp:114:parameter_documentation:WARNING:No matching class parameter for documentation of foreman_proxy_content::pulpcore_allowed_export_path
```
because it's expecting the line I had to delete.  @ehelms what is the correct way forward?